kubernetes/logagent-daemonset.yml

```diff
@@ -1,20 +1,61 @@
-# Kubernetes < v1.9
-# apiVersion: extensions/v1beta1
-# Kubernetes >= v1.9
-# apiVersion: extensions/v1beta2
-# Kubernetes v1.10
-# apiVersion: apps/v1beta2
+# Cluster Role bindings for Logagent
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: sematext-logagent
+  labels:
+    app: sematext-logagent
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: sematext-logagent
+subjects:
+- kind: ServiceAccount
+  name: sematext-logagent
+  namespace: default
+---
+# Cluster Role for Logagent
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sematext-logagent
+  labels:
+    app: sematext-logagent
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - pods
+  verbs:
+  - list
+  - get
+  - watch
+---
+# Service Account for Logagent
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sematext-logagent
+  labels:
+    app: sematext-logagent
+---
+# Daemonset
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: sematext-logagent
+  name: st-logagent
```

Comment:
Why is this `st-logagent` but it's `sematext-logagent` everywhere else? We should keep the naming consistent. Should we use `sematext-logagent` everywhere?

Reply:
Sematext UI instructions use `st-logagent` and the Sematext helm chart uses st-logagent as daemonset name. This change makes the naming consistent with other documentations and instructions.   For the role bindings, sematext/agent uses `sematext-agent` and so we use for role-bindings `sematext-logagent`. If you give all objects the same name, how can users differentiate the different objects?  

Anyhow, we use here the same naming convention as we have it in helm charts already.